it/graphql-multipart/src/test/java/com/linecorp/armeria/graphql/GraphQLTest.java

```diff
@@ -36,7 +36,7 @@
 
 import com.google.common.io.Files;
 
-import com.linecorp.armeria.common.scalar.PathScalar;
+import com.linecorp.armeria.common.scalar.MoreScalars;
```

Comment:
Should the package name be: `com.linecorp.armeria.common.graphql` or `...common.graphql.scalar`? Is it worth having its own package rather than `common.graphql`?

Reply:
Oops... I was missing graphql in my package path. 🤣